gravitee-apim-cypress/cypress/integration/rest-api/apis/import-create-tests.ts

```diff
@@ -0,0 +1,809 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ADMIN_USER, LOW_PERMISSION_USER } from '../../../fixtures/fakers/users/users';
+import {
+  deleteApi,
+  getApiById,
+  getApiMembers,
+  getApiMetadata,
+  importCreateApi,
+} from '../../../commands/management/api-management-commands';
+import { getPage, getPages } from '../../../commands/management/api-pages-management-commands';
+import { ApiImportFakers } from '../../../fixtures/fakers/api-imports';
+import {
+  ApiMetadataFormat,
+  ApiPlanSecurityType,
+  ApiPlanStatus,
+  ApiPlanType,
+  ApiPlanValidationType,
+  ApiPrimaryOwnerType,
+} from '@model/apis';
+import { getPlan } from '../../../commands/management/api-plans-management-commands';
+import { GroupFakers } from '../../../fixtures/fakers/groups';
+import { createGroup, deleteGroup, getGroup } from '../../../commands/management/environment-management-commands';
+import { createUser, deleteUser, getCurrentUser } from '../../../commands/management/user-management-commands';
+import { createRole, deleteRole } from 'commands/management/organization-configuration-management-commands';
+
+context('API - Imports', () => {
+  describe('Create API from import', () => {
+    describe('Create API without ID', () => {
+      let apiId;
+      const fakeApi = ApiImportFakers.api();
+
+      it('should create API and return generated ID', () => {
+        importCreateApi(ADMIN_USER, fakeApi)
+          .ok()
+          .should((response) => {
+            apiId = response.body.id;
+            expect(apiId).to.not.be.null;
+            expect(apiId).to.not.be.empty;
+          });
+      });
+
+      it('should get created API with generated ID', () => {
+        getApiById(ADMIN_USER, apiId)
+          .ok()
+          .should((response) => {
+            expect(response.body.id).to.eq(apiId);
+          });
+      });
+
+      it('should delete created API', () => {
```

Comment:
It might be better to handle prepare & clean-up functionality in before & after hooks. Every it() is counted as a test and we don't really want to test the same thing over and over again. Plus, it creates a wrong impression about the real number of tests.